### ISSUE TITLE
'$env:PATH' in the 'terminal.integrated.env.windows' entry in the setup doesn't seem to work

### ISSUE THREAD

**Issue by ltxhhz** [microsoft/vscode#212062]
Type: <b>Bug</b>

It seems that the current version can't merge the contents of $env:PATH anymore

In version 1.88.1 it looks like this
![image](https://github.com/microsoft/vscode/assets/51697596/487e6b5b-4cc0-4da5-ac86-de1bc64fab15)

Right now in version 1.89.0
![image](https://github.com/microsoft/vscode/assets/51697596/cb022e28-e7fb-4c1b-a902-81dbc81ac8d0)

VS Code version: Code 1.89.0 (b58957e67ee1e712cebf466b995adf4c5307b2bd, 2024-05-01T02:09:22.859Z)
OS version: Windows_NT x64 10.0.18362
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 7735H with Radeon Graphics          (2 x 3194)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: disabled_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|4.00GB (0.81GB free)|
|Process Argv|--crash-reporter-id d162031a-1990-4ad6-ac9b-fe0e9d82e529|
|Screen Reader|no|
|VM|100%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-language-pack-zh-hans|MS-|1.89.2024050109

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vshan820:30294714
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
py29gd2263:30899288
vsclangdf:30486550
c4g48928:30535728
azure-dev_surveyone:30548225
vscrpc:30673769
a9j8j154:30646983
962ge761:30933248
pythongtdpath:30769146
i26e3531:30792625
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30928863
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:30885898
pythontbext0:30879054
accentitlementst:30887150
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
aa_t_chat:30882232
dsvsc019acf:30933242
cp7184t3:30927821

```

</details>

**Comment by meganrogge** [comment]
In your 1.88.1 screenshot, you're using a terminal from the powershell extension. If you create a terminal with that extension in insider's, does it have the env you expect?

**Comment by ltxhhz** [comment]
> In your 1.88.1 screenshot, you're using a terminal from the powershell extension. If you create a terminal with that extension in insider's, does it have the env you expect?

yes,it does.
like this
![image](https://github.com/microsoft/vscode/assets/51697596/90986a75-8d10-410e-b8af-901b911e143f)

**Comment by meganrogge** [comment]
hard to see, but I think that means this is working the same in each version?

**Comment by ltxhhz** [comment]
Hey, I think I misunderstood you.😅
You should be talking about the vscode with the green icon, here's the screenshot
![image](https://github.com/microsoft/vscode/assets/51697596/cf44304e-b282-467e-b6be-99bd79a110fa)

**Comment by ltxhhz** [comment]
@meganrogge Could you please revisit this issue?🙏

**Comment by ltxhhz** [comment]
this issue has been fixed on 1.89.1, thx!
